### DISCUSSION TITLE
求软件

### DISCUSSION THREAD

**Discussion by yuebengyuan** [share121/inter-knot#5427] [其他]
想要一个可以加速歪国软件的加速器～
最好可以加速p站和谷歌网页的。
谢谢
![搜图神器_1727275266434](https://github.com/user-attachments/assets/3f5c1c37-d04f-482e-a8b4-26645e77afdd)

**Comment by ghost** [comment]
这凑合能用但是免费，有付费机场，要吗？
![Screenshot_2024-09-27-13-30-16-367_com bjchuhai](https://github.com/user-attachments/assets/e88154c1-6f1b-428b-bee0-a14dc6839aaf)

**Reply by yuebengyuan** [reply]
是什么加速器？？

**Reply by lingluozhao** [reply]
白鲸加速器

**Comment by hot-ice-seele** [comment]
UU加速器

**Comment by hoyotao** [comment]
我不推荐那些VPN软件，还是机场加代理软件比较好。这边我推荐ikuuu.pw
可以免费用，也有付费版，价格也很实惠。

**Reply by Yizakl** [reply]
谢绝公开分享

**Comment by Hjhnull** [comment]
可以用安心加速器

**Reply by Hjhnull** [reply]
我发现了一个很棒的加速器，海外加速看视频毫无压力
它的下载地址是https://www.axjsq.com
我的邀请码：Bz5JJShb5E，填写我的邀请码即可获得奖励![Screenshot_20240928_075510_com.at.a501014421884997.jpg](https://github.com/user-attachments/assets/d9e28f1c-5e54-4264-9f4d-3ff35293f179)

**Reply by qiyubuhsiqi** [reply]
啊？
![F55D67502117409C1916928C447DD602](https://github.com/user-attachments/assets/cbb774cf-f92d-441e-9e1d-9c76b9809786)

**Comment by 1luik** [comment]
sdkdns

**Comment by cpu644** [comment]
机票，包快的。就是有点小贵
![740c4d52d62d46ba60b46d1a652a6cc](https://github.com/user-attachments/assets/a8da05c5-b586-4f87-abeb-a63a4a69827e)

**Comment by yandujun363** [comment]
如果只是上pixiv可以用nginx反向代理~~只是配置比较麻烦，还要自签CA证书~~

**Comment by AnEwei** [comment]
直接用快连叭

**Comment by Junyxor** [comment]
官网被墙了，12元https://xn--mes358a9urctx.com/#/register?code=nzHSG3ZO可白嫖这个上官网 https://w1.v2free.cc/auth/register?code=fQaP不要做违法的事
